README.md

```diff
@@ -20,13 +20,14 @@ For more information, see: [What is a builder?](https://buildpacks.io/docs/for-p
-|-----------------------------------------------------|---------------------------------------------|---------------------------------------|-------------------|------------------|-------------|
-| [`heroku/buildpacks:18`][buildpacks-tags]           | [`heroku/heroku:18-cnb-build`][heroku-tags] | [`heroku/heroku:18-cnb`][heroku-tags] | 0.16.1            | Shimmed + Native | End-of-life |
-| [`heroku/buildpacks:20`][buildpacks-tags]           | [`heroku/heroku:20-cnb-build`][heroku-tags] | [`heroku/heroku:20-cnb`][heroku-tags] | 0.17.6            | Shimmed + Native | End-of-life |
-| [`heroku/builder-classic:22`][builder-classic-tags] | [`heroku/heroku:22-cnb-build`][heroku-tags] | [`heroku/heroku:22-cnb`][heroku-tags] | 0.17.6            | Shimmed          | End-of-life |
-| [`heroku/builder:20`][builder-tags]                 | [`heroku/heroku:20-cnb-build`][heroku-tags] | [`heroku/heroku:20-cnb`][heroku-tags] | 0.19.3            | Native           | Available   |
-| [`heroku/builder:22`][builder-tags]                 | [`heroku/heroku:22-cnb-build`][heroku-tags] | [`heroku/heroku:22-cnb`][heroku-tags] | 0.19.3            | Native           | Recommended |
+| Builder Image                                       | Base Build Image                            | Base Run Image                        | Lifecycle Version | Buildpack Types  | Status         |
+|-----------------------------------------------------|---------------------------------------------|---------------------------------------|-------------------|------------------|----------------|
+| [`heroku/buildpacks:18`][buildpacks-tags]           | [`heroku/heroku:18-cnb-build`][heroku-tags] | [`heroku/heroku:18-cnb`][heroku-tags] | 0.16.1            | Shimmed + Native | End-of-life    |
+| [`heroku/buildpacks:20`][buildpacks-tags]           | [`heroku/heroku:20-cnb-build`][heroku-tags] | [`heroku/heroku:20-cnb`][heroku-tags] | 0.17.6            | Shimmed + Native | End-of-life    |
+| [`heroku/builder-classic:22`][builder-classic-tags] | [`heroku/heroku:22-cnb-build`][heroku-tags] | [`heroku/heroku:22-cnb`][heroku-tags] | 0.17.6            | Shimmed          | End-of-life    |
+| [`heroku/builder:20`][builder-tags]                 | [`heroku/heroku:20-cnb-build`][heroku-tags] | [`heroku/heroku:20-cnb`][heroku-tags] | 0.19.3            | Native           | Available      |
+| [`heroku/builder:22`][builder-tags]                 | [`heroku/heroku:22-cnb-build`][heroku-tags] | [`heroku/heroku:22-cnb`][heroku-tags] | 0.19.3            | Native           | Recommended    |
+| [`heroku/builder:24`][builder-tags]                 | [`heroku/heroku:24-build`][heroku-tags]     | [`heroku/heroku:24`][heroku-tags]     | 0.19.3            | Native           | In Development |
 
 The builder images above include buildpack support for the following languages: Go, Java, Node.js, PHP, Python, Ruby & Scala. The `heroku/builder-classic:22` builder image variant additionally supports Clojure.
```

Comment:
Do we need to mention ARM64 support somewhere in the README? Sadly the table is already pretty crowded, so maybe it would have to be a mention somewhere in the paragraphs after the table?

Reply:
Yeah, maybe so. I'll add a paragraph. Should we also mention it doesn't support all languages there too?

Reply:
I guess another option might be to skip adding anything to the README for now (like we did for the base-images repo), and only add an entry once we have more languages added?

I was also going to remove the publishing of the legacy builders shortly, and clean up the readme and table at the same time (eg remove the native vs shimmed column), which might free up some table space for a "Supported Architectures" column etc?

Reply:
I'm fine with any of the above tbh (incl just leaving as it is in the current version of the PR) - whatever you prefer :-)

Reply:
Yeah, I guess I'll leave it as-is for now. I look forward to the cleanup where we can remove several other sources of confusion.